dist/README.md

```diff
@@ -36,25 +36,32 @@ You can grab the relevant MathJax files in `./dist/extras/mathjax/`.
 ### To include localization
 
 Plotly.js defaults to US English (en-US) and includes British English (en) in the standard bundle.
-Many other localizations are available - here is an example using Swiss-German (de-CH),
-see the contents of this directory for the full list.
-They are also available on our CDN as https://cdn.plot.ly/plotly-locale-de-ch-latest.js OR https://cdn.plot.ly/plotly-locale-de-ch-1.38.1.js
-Note that the file names are all lowercase, even though the region is uppercase when you apply a locale.
+This example uses Swiss-German (de-CH), but many other localizations are available (see the contents of this directory for the full list). All available locales have working date localizations, but only a subset will localize on-graph text (e.g., modebar controls). See [here](https://github.com/plotly/plotly.js/pulls?q=is%3Apr+label%3A%22type%3A+translation%22+is%3Aclosed) for a list of "fully-supported" locales.
```

Comment:
Perhaps even better, we could [ask translation authors](https://github.com/plotly/plotly.js/blob/master/.github/PULL_REQUEST_TEMPLATE.md#translations) to add an entry in a table of "fully" supported localizaion instead of linking to a list of PRs/